hnn_core/tests/test_drives.py

```diff
@@ -33,7 +30,9 @@ def test_external_drive_times():
 
     # validate poisson input time interval
     drive_type = 'poisson'
-    dynamics = drive_specs['extpois']['dynamics']
+    dynamics = {'tstart': 0, 'tstop': 250.0, 'rate_constant':
+                {'L2_basket': 1, 'L2_pyramidal': 140.0, 'L5_basket': 1,
+                 'L5_pyramidal': 40.0}}
```

Comment:
The only other thing I did was to try out my suggestion from earlier where we explicitly define an instance of the `dynamics` dict instead of reading one in from a gamma param file.

Reply:
Oh that's perfect! Definitely agree it's much less round about tested to eat